zanata-war/src/main/webapp/WEB-INF/pages.xml

```diff
@@ -111,14 +14,7 @@
     </navigation>
   </page>
 
-  <page view-id="/account/register.xhtml">
-    <navigation from-action="#{identity.isLoggedIn()}">
-      <rule if-outcome="true">
-        <redirect view-id="/dashboard/home.xhtml" />
-      </rule>
-    </navigation>
-  </page>
-
+  <!-- TODO [CDI] uses conversation scope facesMessages. if migrated to faces-config, it will lose notification message -->
```

Comment:
JSF's flash scope in theory is designed to solve this problem (post -> redirect -> show the message). Sean thinks the deltaspike ViewAccessScope (as a replacement for conversation scope) may work.